tests/transforms/test_split_non_commuting.py

```diff
@@ -0,0 +1,241 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Tests for the transform ``qml.transform.split_non_commuting()`` """
+import pytest
+import numpy as np
+import pennylane as qml
+import pennylane.numpy as pnp
+
+from pennylane.transforms import split_non_commuting
+
+
+
+
+# Unit tests for split_non_commuting
```

Comment:
Not mandatory, but these comments --- which seem to be used to categorize the test functions --- indicate that test classes for organization could be a good fit here :)